src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -94,6 +94,7 @@
                     + "                                         ERROR,WARN,INFO,DEBUG,TRACE [default: INFO].\n"
                     + " -Q, --quickcheck-test-mode              Run in QuickCheck test mode\n"
                     + " -P <seconds>, --cm-poll-interval        Configuration manager poll interval [default: 1]\n"
+                    + " -v, --verify                            Compute and verify checksums for log entries.\n"
```

Comment:
Hm,  the option should be -n --no-verify --- as in, checksums should be on by default. 

Which I guess raises the question, what happens if the user runs with checksums off, then restarts the server with checksums on? I think the behavior should be that log entries without checksums are still considered valid.